tests/click_tests/test_recovery.py

```diff
@@ -0,0 +1,67 @@
+import pytest
```

Comment:
this is run by normal python

python itself doesn't need `__init__.py` anymore, but pytest uses it to discover top-level package IIUC. So if you leave out the `__init__.py` here, then `tests/click_tests` is set as the root of `PYTHONPATH`, instead of just `tests`. and that breaks relative imports

Reply:
> this is run by normal python

Sure :man_facepalming:.

----

Hmm interesting. Okay thanks.